src/Roslyn.Diagnostics.Analyzers/Core/CreateTestAccessor.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System.Collections.Immutable;
+using System.Linq;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.Diagnostics;
+
+namespace Roslyn.Diagnostics.Analyzers
+{
+    [DiagnosticAnalyzer(LanguageNames.CSharp, LanguageNames.VisualBasic)]
+    public class CreateTestAccessor : DiagnosticAnalyzer
+    {
+#pragma warning disable CA1802 // Use literals where appropriate
+        public static readonly string GetTestAccessorMethodName = "GetTestAccessor";
+        public static readonly string TestAccessorTypeName = "TestAccessor";
```

Comment:
Is it an explicit choice to not have these fields `const`? If so it might be worth adding a comment.

Reply:
Will change to properties